data/quests.ts

```diff
@@ -1,6 +1,109 @@
 import { ListType } from "./types";
 
 export const Quests: Array<ListType> = [
+  /* TEMPLATE     
+  {
+    // Can be left blank to be filled later
+    id: "",
+
+    // Name of the NPC/ending
+    name: "",
+
+    requirements: [
+      {
+        // Can be left blank to be filled later
+        id: "",
+
+        // Short sentence in brief describing necessary actions with few details
+        // Acceptable details:
+        // - Nearby Site of Grace, or Map Section
+        // - Necessary bosses and items
+        // - Dialog options
+        description: ""
+      },
+    ]
+  },
+  */
+  {
+    id: "",
+    name: "Kenneth Haight (Incomplete)",
+    requirements: [
+      {
+        id: "",
+        description: "Talk to Kenneth in West Limgrave near Third Church of Marika site of grace.",
+      },
+      {
+        id: "",
+        description: "Clear out enemies in Fort Haight in West Limgrave.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at his first location. Receive a reward.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at Fort Haight.",
+      },
+      {
+        id: "",
+        description: "To Be Determined ...",
```

Comment:
I'd say it's worth to include what is known about this quest line after patch 1.03.
```suggestion
        id: "",
        description: "Once you have completed Nepheli's questline, talk to Kenneth at the Stormveil Castle throne room.",
```